apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -131,16 +145,20 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     event ChangeOverruleWindow(uint64 overruleWindow);
     event ChangeQuietEndingConfiguration(uint64 quietEndingPeriod, uint64 quietEndingExtension);
 
+    // Note: may want to also emit the evm script
     event StartVote(uint256 indexed voteId, address indexed creator, bytes context);
     event PauseVote(uint256 indexed voteId, uint256 indexed challengeId);
     event ResumeVote(uint256 indexed voteId);
     event CancelVote(uint256 indexed voteId);
     event ExecuteVote(uint256 indexed voteId);
+    // Note: maybe we should stick with Vote* or *Vote? E.g. ExtendVote()?
     event VoteQuietEndingExtension(uint256 indexed voteId, bool passing);
 
+    // Note: should we remove stake?
     event CastVote(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
     event ChangeRepresentative(address indexed voter, address indexed representative);
     event ProxyVoteFailure(uint256 indexed voteId, address indexed voter, address indexed representative);
+    // Note: maybe we can roll this into CastVote if we add the caster there?
     event ProxyVoteSuccess(uint256 indexed voteId, address indexed voter, address indexed representative, bool supports);
```

Comment:
Was wondering if we really need to emit this event—CastVote could be emitted with the `caster` to provide the same information as this (saving us 2-3k gas per proxy vote)

Reply:
Addressed in fa29e27058b852b6ec20115a4089316212db66ca